src/ripple/app/misc/impl/ValidatorSite.cpp

```diff
@@ -74,20 +98,16 @@ ValidatorSite::load (
 
     for (auto uri : siteURIs)
     {
-        parsedURL pUrl;
-        if (! parseUrl (pUrl, uri) ||
-            (pUrl.scheme != "http" && pUrl.scheme != "https"))
+        try
+        {
+            sites_.emplace_back (uri);
```

Comment:
As-is, this makes an unneeded copy. Changing the for loop param to `auto const&` will fix this.. (We dn't want to `std::move` here since `uri` is used in the catch block).